scripts/generate_insolar_configs.go

```diff
@@ -223,7 +225,8 @@ func main() {
 		pulsarConfig.Pulsar.PulseDistributor.BootstrapHosts = append(pulsarConfig.Pulsar.PulseDistributor.BootstrapHosts, node.Host)
 	}
 
-	writeInsolarConfigs(insolarConfigs)
+	writeInsolarConfigs(outputDir, discoveryNodesConfigs)
+	writeInsolarConfigs(outputDir+"/nodes", nodesConfigs)
```

Comment:
use https://golang.org/pkg/path/filepath/ join to work with paths

Reply:
fixed